2_Growth-Maturity-Decline.md

```diff
@@ -1,79 +0,0 @@
-
-Metric | Question
---- | ---
-[Open Issues](activity-metrics/open-issues.md) | What is the number of open issues?
-[Closed Issues](activity-metrics/closed-issues.md) | What is the number of closed issues?
-[Issue Resolution Efficiency](activity-metrics/issue-resolution-efficiency.md) | What is the number of closed issues/number of abandoned issues?
-[Open Issue Age](activity-metrics/open-issue-age.md) | What is the the age of open issues?
-[First Response to Issue Duration](activity-metrics/first-response-to-issue-duration.md) | What is the duration of time for a first response to an issue?
-[Closed Issue Resolution Duration](activity-metrics/closed-issue-resolution-duration.md) | What is the duration of time for issues to be resolved?
-[Issue Resolution Efficiency](activity-metrics/issue-resolution-efficiency.md) |  What is the ratio of closed issues/number of abandoned issues?
-
-
-## Code Development
-
-Goal: Identify how effective the community is at merging new code into the codebase.
-
-Metric | Question
---- | ---
-[Code Commits](activity-metrics/code-commits.md) | What is the number of code commits?
-[Lines of Code Changed](activity-metrics/lines-of-code-changed.md) | What is the number of lines of code changed?
-[Code Reviews](activity-metrics/code-reviews.md) | What is the number of code reviews?
-[Code Merge Duration](activity-metrics/code-merge-duration.md) | What is the duration of time between code merge request and code commit?
-[Code Review Efficiency](activity-metrics/code-review-efficiency.md) | What is the number of merged code changes/number of abandoned code change requests?
-[Maintainer Response to Merge Request Duration](activity-metrics/maintainer-response-to-merge-request-duration.md) | What is the duration of time for a maintainer to make a first response to a code merge request?
-[Code Review Iteration](activity-metrics/code-review-iteration.md) | What is the number of iterations that occur before a merge request is accepted or declined?
-[Forks](activity-metrics/forks.md) | Forks are a concept in distributed version control systems like GitHub. It is a proxy for the approximate number of developers who have taken a shot at building and deploying the codebase *for development*.
-[Pull Requests Open](activity-metrics/pull-requests-open.md) | Number of open pull requests.
-[Pull Requests Closed](activity-metrics/pull-requests-made-closed.md) | Number of closed pull requests.
-[Pull Request Comment Duration](activity-metrics/pull-requests-comment-duration.md) | The difference between the timestamp of the pull request creation date and the most recent comment on the pull request.
-[Pull Request Comment Diversity](activity-metrics/pull-requests-comment-diversity.md) | Number of each people discussing each pull request.
-[Pull Request Comments](activity-metrics/pull-request-comments.md) | Number of comments on each pull request. 
-
-
-[](activity-metrics/)
-
-## Community Growth

```

Comment:
I think what you are getting at is providing a summary of the GMD metrics as a top level page, yes?

Reply:
Agreed, these changes may be premature.

Reply:
And yes, part of the pull request does do that.